### PR TITLE
Publishing script Maven Central

### PR DESCRIPTION
**Pull request by askur87**

### CODE REVIEW

File: uikit/src/main/java/com/midtrans/sdk/uikit/activities/PaymentMethodsActivity.java [hunk 1]
```diff
@@ -332,6 +332,7 @@ public void onFailure(Token token, String reason) {
 
                 @Override
                 public void onError(Throwable error) {
+                    enableButtonBack(true);
```
Comment (theanh0512) [inline uikit/src/main/java/com/midtrans/sdk/uikit/activities/PaymentMethodsActivity.java:335]: why is this change here?

Comment (askur87) [reply uikit/src/main/java/com/midtrans/sdk/uikit/activities/PaymentMethodsActivity.java:335]: this is needed to fix the issue can't press the back button on the error screen, I just fix some issue on this branch also

**Review by theanh0512 — APPROVED** [review]

File: build-system/corekit/publishing.gradle [hunk 2]
```diff
@@ -1,5 +1,6 @@
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'maven-publish'
+apply plugin: 'signing'
```
Comment (uziwuzzy) [inline build-system/corekit/publishing.gradle:3]: Sorry I am not sure about what the signing do, is this required for uploading to maven? will other dev able to use the signed sdk library?

File: build-system/corekit/publishing.gradle [hunk 3]
```diff
@@ -12,114 +13,174 @@ task sourceJar(type: Jar) {
...
+        }
     }
-    publications = ['BintrayProduction', 'BintraySandbox', 'BintrayUat']
-}
\ No newline at end of file
+}
+
+signing {
+    sign publishing.publications
+}
```
Comment (uziwuzzy) [inline build-system/corekit/publishing.gradle:186]: Sorry I am not sure about what the signing do, is this required for uploading to maven? will other dev able to use the signed sdk library?

File: build-system/corekit/publishing.gradle (hunk 2 above)
Comment (askur87) [reply build-system/corekit/publishing.gradle:3]: yes, signing is required by maven, others can do but need to register to maven account. you guys need to register and set up the credential, will share how it's works later.

File: build-system/corekit/publishing.gradle (hunk 3 above)
Comment (askur87) [reply build-system/corekit/publishing.gradle:186]: same comment above

**Review by uziwuzzy — APPROVED** [review]
